jenkinsfiles/kubernetes-upstream.Jenkinsfile

```
@@ -86,8 +86,12 @@ pipeline {
                 timeout(time: 150, unit: 'MINUTES')
             }
 
+            environment {
+                POLL_TIMEOUT_SECONDS=150
+            }
+
             steps {
-                sh 'cd ${TESTDIR}; vagrant ssh k8s1-${K8S_VERSION} -c "cd /home/vagrant/go/${PROJ_PATH}; sudo ./test/kubernetes-test.sh ${DOCKER_TAG}"'
+                sh 'cd ${TESTDIR}; vagrant ssh k8s1-${K8S_VERSION} -c "cd /home/vagrant/go/${PROJ_PATH}; env pollTimeout=${POLL_TIMEOUT_SECONDS} sudo ./test/kubernetes-test.sh ${DOCKER_TAG}"'
```

```suggestion
                sh 'cd ${TESTDIR}; vagrant ssh k8s1-${K8S_VERSION} -c "cd /home/vagrant/go/${PROJ_PATH}; sudo ./test/kubernetes-test.sh ${DOCKER_TAG}"'
```

Why is it necessary to have the `env pollTimeout=${POLL_TIMEOUT_SECONDS}` part? Why can't you use the `POLL_TIMEOUT_SECONDS` directly in the script?

---

Sorry for the delay in reply. This was my first time working with Jenkins. I misunderstood that those variables need to be explicitly passed to the scripts. I'll just fix this.